mongo/countopt/etimatedcountopt.go

```diff
@@ -0,0 +1,60 @@
+package countopt
+
+import (
+	"github.com/mongodb/mongo-go-driver/core/option"
+)
+
+var estimatedDocumentCountBundle = new(EstimatedDocumentCountBundle)
+
+// EstimatedDocumentCount is options for the estimatedDocuemntCount() function
+type EstimatedDocumentCount interface {
+	estimatedCount()
+	ConvertEstimateDocumentCountOption() option.EstimatedDocumentCountOptioner
+}
+
+// EstimatedDocumentCountBundle is a bundle of EstimatedDocumentCount options
+type EstimatedDocumentCountBundle struct {
```

Comment:
This type is missing an Unbundle method.

Reply:
Done.